scripts/test-remote-integration.sh

```diff
@@ -60,7 +60,6 @@ else
 
 	# Configuration for more verbose output
 	export VSCODE_CLI=1
-	export ELECTRON_ENABLE_STACK_DUMPING=1
```

Comment:
Does the removal of this flag also have to do with crash reporter changes?

Reply:
Nope it is unrelated, the flag is useful when the crash reporter client is disabled and we want to print stack traces from renderer process to stderr. In our CI we are uploading crash dumps generated by crash reporter as artifacts and hence this variable is not required.